rundmcmc/ingest.py

```diff
@@ -38,15 +35,16 @@ def ingest(filepath, name_of_geoid_col):
             # entry to j (to represent actual adjacency).
             if adj == 1:
                 row.append(df["perimeter"][j])
-                adjacency[i][j] = j
+                adjacency[i][j] = int(j)
 
         perims.append(row)
 
     # Strip out zeros from adjacency list.
     for row in adjacency:
-        neighbors.append([i for i in row if i != 0])
+        neighbors.append([i for i in row if type(i) is int])
```

Comment:
Are we no longer stripping out zeros? If so, then the above comment it incorrect now.

Reply:
We are still ripping out zeroes.